### DISCUSSION TITLE
Oracle Explorer in Positron vs in VS Code

### DISCUSSION THREAD

**Discussion by JoeCrozier** [posit-dev/positron#4032] [Q&A]
I've been using the Oracle Explorer extension in VS Code, and that extension also exists in Positron, but I cant get the same connection to work, with the exact same connection details.  
Vs Code:
![image](https://github.com/user-attachments/assets/a087d37e-6268-405c-bfd5-cf5c179cf545)

Positron:
![image](https://github.com/user-attachments/assets/0c31a8de-703e-43dd-b202-841f9bf08412)

(Connection name is different in this screenshot but I've done it with an identical connection name.  And the password that is anonymized for this screenshot was copied and pasted into both.

The error:
![image](https://github.com/user-attachments/assets/7b4e8d61-bff5-412c-891f-c8c66ef166dd)

That folder is completely empty.  The extension settings are also identical in both programs, both of them point to that folder. Any idea whats wrong?  Seems like it must be a positron thing but I have no idea where to begin.

**Comment by simonpcouch** [comment]
Thanks for the post! Before I spend some time debugging further, I did notice that as well as the Connection name changing like you mention, the User name (`NP` to `PROD`) and `SERVICE_NAME` ( `pdblhs` to `pdblhst`) change. Is it possible that that's responsible for the failure to connect?
